esmvaltool/cmorizers/data/formatters/datasets/esacci_lst.py

```diff
@@ -1,160 +1,138 @@
+                                                  attributes=None,
+                                                  coord_system=None,
+                                                  circular=False
+                                                  )
+
+                cubes.attributes = {}
+                cubes.attributes['var'] = var
+
+                try:
+                    cubes.remove_coord('time')
+                except:
+                    logger.info('Coord fix issue %s' % cubes.long_name)
+
+                cubes.add_dim_coord(time_coord, 0)
+
+                if cubes.long_name == 'land surface temperature':
+                    cubes.long_name = 'surface_temperature'
+                    cubes.standard_name = 'surface_temperature'
+
+                try:
+                    cubes = fix_coords(cubes)
+                except:
+                    logger.info('skip fixing')
+                    logger.info(cubes.long_name)
+
+                # this is needed for V1 data, V3 data is ok
+                try:
+                    cubes.coords()[2].standard_name = 'longitude'
+                except:
+                    # No change needed
+                    pass
+
+                var_name = cubes.attributes['var']
+
+                if cubes.var_name == 'lst':
+                    cubes.var_name = 'ts'
+
+                if 'Day' in var_name:
+                    cubes.long_name += ' Day'
+                    cubes.var_name += '_day'
+
+                if 'Night' in var_name:
+                    cubes.long_name += ' Night'
+                    cubes.var_name += '_night'
+
+                # Land cover class gives this error when
+                # loading in CMORised files
+                # OverflowError:
+                # Python int too large to convert to C long error
+                # This fixes it:
+                if 'land cover' in cubes.long_name:
+                    cubes.data.fill_value = 0
+
+                    # Get rid of anything outide 0-255
+                    cubes.data[np.where(cubes.data < 0)] = 0
+                    cubes.data[np.where(cubes.data > 255)] = 0
+
+                    cubes.data = cubes.data.filled()
+
+                    # This is the line the ultimately solves things.
+                    # Will leave the other checks above in because they
+                    cubes.data = cubes.data * 1.0
+
+                save_name = f'{out_dir}/OBS_ESACCI-LST_sat_3.00_Amon_' + \
+                            f'{var_name}_{year}{month:02d}.nc'
+                iris.save(cubes,
+                          save_name
+                          )
+
+
+def load_cubes(in_dir, file, year, month, variable_list):
+    """Load files into cubes based on variables wanted in variable_list."""
+    logger.info(f'Loading {in_dir}/{file}{year}{month:02d}.nc')
+    cube = iris.load_cube(f'{in_dir}/{file}{year}{month:02d}*.nc',
+                          variable_list
+                          )
```

Comment:
your code formatting is not following standard rules, please run a `yapf -i` on your codes when you done editing